src/main/java/com/welcome/bot/controllers/UserController.java

```diff
@@ -57,18 +64,38 @@
     private UserSettingsRepository userSettingsRepository;
     @Autowired
     MessageSource messageSource;
+    @Autowired
+    InviteRepository inviteRepository;
```

Comment:
Scope should be private not package-private.

Reply:
Resolved.